app/code/Magento/GiftMessageGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,20 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Cart {
+    gift_message: GiftMessage @resolver (class: "\\Magento\\GiftMessageGraphQl\\Model\\Resolver\\GiftMessage") @doc(description: "The entered gift message for the cart")
+}
+#TODO add resolver for SalesItemInterface
+type SalesItemInterface {
```

Comment:
Hi @rogyar . 
 I confused about implementing this point.  
I can't understand which GraphQl query should be for testing this one.
Maybe you could clarify it.
Thanks a lot.